jobs/build/aws-ami/Jenkinsfile

```diff
@@ -25,7 +25,7 @@ properties(
                                                     openshift-enterprise      Openshift Enterpriseonline <br>
                                     ''',
                                     name: 'DEPLOYMENT_TYPE'],
-                                 [$class: 'hudson.model.StringParameterDefinition', defaultValue: 'opstest', description: 'Clusterid to set for ami build. Also used for VPC and other settings', name: 'CLUSTERID'],
+                                 [$class: 'hudson.model.StringParameterDefinition', defaultValue: 'default', description: 'Clusterid to set for ami build. Also used for VPC and other settings', name: 'CLUSTERID'],
                                  [$class: 'hudson.model.StringParameterDefinition', defaultValue: '3.7.0', description: 'Release version (matches version in branch name for release builds)', name: 'RELEASE_VERSION'],
                                  [$class: 'BooleanParameterDefinition', defaultValue: false, description: 'Mock run to pickup new Jenkins parameters?.', name: 'MOCK'],
                                  [$class: 'hudson.model.StringParameterDefinition', defaultValue: 'ami-068a787c', description: 'Base AMI id to build from.', name: 'BASE_AMI_ID'],
```

Comment:
If the job is supposed to be part of the build process, how do we determine
which AMI to use as a base?

Reply:
This is a good question.  I hate to hard code it because if someone comes along and updates the base AMI to have the latest errata or updates we need to build on top of it.

Is there a way to force someone to pass in the AMI id? I defaulted in my test but I'm not sure we want a default.  If a security vulnerability is found in our default AMI or a mistake happens then we never want to build off this default.  We probably need to be told from which AMI we need to build on top of.  For this reason I placed it as a parameter.

The new current AMI should be updated as I built a recent, clean RHEL 7.4 image.

Reply:
We can make it a required parameter, for sure.  But if we want this jobs to be
part of the build process, that only pushes the problem up the stack: the job
that will call this one will have to know which AMI to pass as a parameter.

If we don't have a way to programmatically find the name of the image, then
making it a parameter of the (main) build job might be a solution.

@jupierce: would that be a problem?